x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/bulk_create_cases.ts

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type SuperTest from 'supertest';
+import expect from '@kbn/expect';
+import { BulkCreateCasesResponse } from '@kbn/cases-plugin/common/types/api';
+import { CaseSeverity } from '@kbn/cases-plugin/common';
+import { CaseStatuses, CustomFieldTypes } from '@kbn/cases-plugin/common/types/domain';
+import { User } from '../../../../common/lib/authentication/types';
+import { defaultUser, getPostCaseRequest, postCaseResp } from '../../../../common/lib/mock';
+import {
+  deleteAllCaseItems,
+  getCaseUserActions,
+  getSpaceUrlPrefix,
+  removeServerGeneratedPropertiesFromCase,
+  removeServerGeneratedPropertiesFromUserAction,
+} from '../../../../common/lib/api';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import {
+  secOnly,
+  secOnlyRead,
+  globalRead,
+  obsOnlyRead,
+  obsSecRead,
+  noKibanaPrivileges,
+  testDisabled,
+  superUser,
+} from '../../../../common/lib/authentication/users';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  /**
+   * There is no official route that supports
+   * bulk creating cases. The purpose of this test
+   * is to test the bulkCreate method of the cases client in
+   * x-pack/plugins/cases/server/client/cases/bulk_create.ts
+   *
+   * The test route is configured here x-pack/test/cases_api_integration/common/plugins/cases/server/routes.ts
+   */
+  describe('bulk_create_cases', () => {
```

Comment:
Maybe we could include a test for the assignees/platinum license logic. I also didn't see any in `x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/post_case.ts`.

Reply:
I thought the unit test would be sufficient. Do you prefer an integration test?